.github/workflows/release.yml

```diff
@@ -44,4 +44,44 @@ jobs:
         run: |
           echo "${{ steps.lt_release.outputs.releasability }}"
           echo "${{ steps.lt_release.outputs.release }}"
-      
+  maven-central-sync:
```

Comment:
Some env vars needs to be delete
BINTRAY_USER: ${{ secrets.BINTRAY_USER }}
BINTRAY_TOKEN: ${{ secrets.BINTRAY_TOKEN }}
CENTRAL_USER: ${{ secrets.CENTRAL_USER }}
CENTRAL_PASSWORD: ${{ secrets.CENTRAL_PASSWORD }}
GPG_PASSPHRASE: ${{ secrets.GPG_PASSPHRASE }}